teps/0079-tekton-catalog-support-tiers.md

```diff
@@ -523,24 +537,380 @@ general oversight. Contributors are already hosting Catalogs in their own organi
+
+We propose to use the [kind-e2e task][kind-e2e] to spin up a `KinD` cluster to run the test `pipelines` when triggered. The `kind-e2e task` executes the `./test/e2e-tests.sh` script in the target catalog which then runs the test `pipelines` in the catalog resource directories(`./{resource-type}/{resource-name}/tests/pipelineRun.yaml`). The `Tekton Pipelines` versions to be tested against can be configured in the `./test/e2e-tests.sh` script, which will be further discussed in the below sections.
+
+Please refer to the [kind-e2e doc][kind-e2e-doc] for more information.
+
+###### Integration Test on PR Merge
+
+We propose to test the Verified Catalogs against:
+- The **most recent** `Tekton Pipelines` release 
+- The **minimum compatible** `Tekton Pipelines` release (specified by `tekton.dev/minVersion` label of the Task)
+
+The successful completion of the integration tests passes the `pull-tekton-catalog-integration-tests-latest` and `pull-tekton-catalog-integration-tests-min-compatible` GitHub checks respectively to unblock a PR merge. The integration test should be triggered by pull request `opened`, `reopened`, `synchronize` events, or by `/test pull-tekton-catalog-integration-tests-latest` and `/test pull-tekton-catalog-integration-tests-min-compatible` comments in the pull request.
+
+
+This is a conceptual high-level flow:
+
+  ```
+           |                                      |
+           v                                      v
+  start-integration-test-GitHub-check       git-batch-merge
+                                                  |
+                                                  v
+                                            kind-e2e-test
+                                                  |
+                                                  v      
+                                  update-integration-test-GitHub-check
+                                                  |
+                                                  v     
+                                            upload-logs                                                                          
+  ```
+
+###### Nightly Integration Test
+
+We propose to test the Verified Catalogs against the **latest nightly release** of `Tekton Pipelines` as periodic nightly jobs. 
+
+The pro and con of this approach are the same: we may catch issues with Tekton Pipelines itself. Since backward incompatible changes require an ApiGroup bump, this should never fail, but probably sometimes will. Failures here are probably caused by bugs in Tekton Pipelines. This means we may catch more issues before releases, but also means that Catalog tests may fail for non-catalog related reasons.
+
+Since these failures are more likely to be caused by Tekton Pipelines than by the Tasks, we will only run these as periodic nightly jobs and not let these failures block a PR merge. It will be up to Tekton Catalog Maintainers to track down their failures.
```

Comment:
can we send slack notifications to maintainers? wondering how the tekton pipelines maintainers can be aware of the failures in nightly releases that are caught in the nightly catalog tests

Reply:
SGTM! I have updated this section. PTAL!